wheels/build/wheels.yml

```diff
@@ -124,10 +124,9 @@ packages:
             - cyrus-sasl-devel
             - libidn-devel
     python_lzo:
-        version: 1.8
+        version: 1.11
         src:
-            - http://www.oberhumer.com/opensource/lzo/download/LZO-v1/python-lzo-1.08.tar.gz
```

Comment:
Should I replace this with https://pypi.python.org/packages/bc/ac/cec925b9367b8f7f18a606d462fe17069fb13b0fd2fe7df0c324a83a8018/python-lzo-1.11.tar.gz ?

Reply:
You should be able to drop this line entirely since the source is in PyPI. I believe it wasn't before.